pandas/tools/plotting.py

```diff
@@ -560,14 +575,14 @@ def f(x):
     for i in range(n):
```

Comment:
Oh, of course. I'll do that and the `asv` benchmark. Will probably be over the weekend.